docs/v0.3.X/mkdocs.yml

```diff
@@ -53,4 +53,5 @@ nav:
     - Inspect configurations: cscli/cscli_inspect.md
     - Manage simulation: cscli/cscli_simulation.md
     - Dashboard: cscli/cscli_dashboard.md
+  - Upgrade V0.X to V1.X: migration.md
```

Comment:
I think we might want to add it as well to v1.X doc

Reply:
Does this documentation should be at the top level ?

Reply:
I think there are several opinions on this, we need to discuss quickly and take a decision.